appveyor.yml

```diff
@@ -10,11 +10,15 @@ environment:
   TESTS: "false"
 
 install:
-  - appveyor DownloadFile https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/appveyor-opam.sh
+  - appveyor DownloadFile https://dl.dropboxusercontent.com/s/eo4igttab8ipyle/opam32.tar.xz
```

Comment:
We probably should find somewhere else to keep this.  I got some download errors when trying out this branch.